snapshot/testsuite.go

```diff
@@ -0,0 +1,89 @@
+package snapshot
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/docker/containerd"
+	"github.com/docker/containerd/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+// Create a New Layer on top of base layer with Prepare, Stat on new layer, should return Active layer.
+func checkSnapshotterStatActive(t *testing.T, snapshotter Snapshotter, work string) {
+	ctx := context.TODO()
+	preparing := filepath.Join(work, "preparing")
+	if err := os.MkdirAll(preparing, 0777); err != nil {
+		t.Fatal(err)
+	}
+
+	mounts, err := snapshotter.Prepare(ctx, preparing, "")
+	if err != nil {
```

Comment:
(multiple occurrences) We're already using `github.com/stretchr/testify/assert`, but we could also start using `github.com/stretchr/testify/require` too.  Then this would be as simple as `require.Nil(t,err)`.  ([`require`](https://godoc.org/github.com/stretchr/testify/require) is the same as `assert` except that it does `t.Fatal` instead of `t.Error`.)